modules/user_passwords/scripts/update.ts

```diff
@@ -0,0 +1,50 @@
+import { Empty, RuntimeError, ScriptContext } from "../module.gen.ts";
+import { ALGORITHM_DEFAULT, Algorithm, hash } from "../utils/common.ts";
+
+export interface Request {
```

Comment:
should we add a public script to update the password that requires the current password?